### ISSUE TITLE
fatal: repository 'https://git.ruby-lang.org/ruby.git/' not found

### ISSUE THREAD

**Issue by yahonda** [mame/rubyfarm-bisect#2]
`rubyfarm-bisect` shows "fatal: repository 'https://git.ruby-lang.org/ruby.git/' not found" error unless `-u` option.  

### Steps to reproduce:

```ruby
% ruby -v
% gem install rubyfarm-bisect
% rubyfarm-bisect -g c55db6aa271df4a689dc8eb0039c929bf6ed43ff -b 02aadf1032d71a3f81653a955f2635c8853b0aca ruby -e 'p /re/.match?(nil)'
```

### Actual result:
- It gets "fatal: repository 'https://git.ruby-lang.org/ruby.git/' not found".

```ruby
% ruby -v
ruby 2.7.1p83 (2020-03-31 revision a0c7c23c9c) [x86_64-darwin19]
% gem install rubyfarm-bisect
Successfully installed rubyfarm-bisect-1.1.2
Parsing documentation for rubyfarm-bisect-1.1.2
Done installing documentation for rubyfarm-bisect after 0 seconds
1 gem installed
% rubyfarm-bisect -g c55db6aa271df4a689dc8eb0039c929bf6ed43ff -b 02aadf1032d71a3f81653a955f2635c8853b0aca ruby -e 'p /re/.match?(nil)'
color legend: git clone git bisect rubyfarm-bisect docker test script

Cloning into 'ruby'...
fatal: repository 'https://git.ruby-lang.org/ruby.git/' not found

Traceback (most recent call last):
	7: from /Users/yahonda/.rbenv/versions/2.7.1/bin/rubyfarm-bisect:23:in `<main>'
	6: from /Users/yahonda/.rbenv/versions/2.7.1/bin/rubyfarm-bisect:23:in `load'
	5: from /Users/yahonda/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/gems/rubyfarm-bisect-1.1.2/exe/rubyfarm-bisect:58:in `<top (required)>'
	4: from /Users/yahonda/.rbenv/versions/2.7.1/lib/ruby/2.7.0/tmpdir.rb:89:in `mktmpdir'
	3: from /Users/yahonda/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/gems/rubyfarm-bisect-1.1.2/exe/rubyfarm-bisect:59:in `block in <top (required)>'
	2: from /Users/yahonda/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/gems/rubyfarm-bisect-1.1.2/exe/rubyfarm-bisect:59:in `chdir'
	1: from /Users/yahonda/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/gems/rubyfarm-bisect-1.1.2/exe/rubyfarm-bisect:63:in `block (2 levels) in <top (required)>'
/Users/yahonda/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/gems/rubyfarm-bisect-1.1.2/exe/rubyfarm-bisect:63:in `chdir': No such file or directory @ dir_chdir - ruby (Errno::ENOENT)
%
```

### Workaround:

- clone Ruby repository like GitHub and use -u option

```ruby
% git clone https://github.com/ruby/ruby.git
% rubyfarm-bisect -u /path/to/ruby -g 02aadf1032d71a3f81653a955f2635c8853b0aca -b b563439274a402e33541f5695b1bfd4ac1085638 ruby -e 'p /re/.match?(nil)'
```

### Result with a workaround:
* I have cloned Ruby repository to `/Users/yahonda/src/github.com/ruby/ruby`

```ruby
% rubyfarm-bisect -u /Users/yahonda/src/github.com/ruby/ruby -g c55db6aa271df4a689dc8eb0039c929bf6ed43ff -b 02aadf1032d71a3f81653a955f2635c8853b0aca ruby -e 'p /re/.match?(nil)'

... snip ...
2a22a6b2d8465934e75520a7fdcf522d50890caf is the first bad commit
commit 2a22a6b2d8465934e75520a7fdcf522d50890caf
Author: Kenichi Kamiya <kachick1@gmail.com>
Date:   Thu Oct 17 17:44:46 2019 +0900

    Regexp#match{?} with nil raises TypeError as String, Symbol (#1506)

    * {String|Symbol}#match{?} with nil returns falsy

    To improve consistency with Regexp#match{?}

    * String#match(nil) returns `nil` instead of TypeError
    * String#match?(nil) returns `false` instead of TypeError
    * Symbol#match(nil) returns `nil` instead of TypeError
    * Symbol#match?(nil) returns `false` instead of TypeError

    * Prefer exception

    * Follow empty ENV

    * Drop outdated specs

    * Write ruby/spec for above

    https://github.com/ruby/ruby/pull/1506/files#r183242981

    * Fix merge miss

 re.c                                |  2 +-
 spec/ruby/core/regexp/match_spec.rb | 40 ++++++++++++++++++++++++++-----------
 test/ruby/test_regexp.rb            |  4 ++--
 test/ruby/test_string.rb            |  5 +++++
 test/ruby/test_symbol.rb            |  5 +++++
 5 files changed, 41 insertions(+), 15 deletions(-)
bisect run success
```

**Comment by yahonda** [comment]
It simply reproduces with `git clone`. 

```
% git clone https://git.ruby-lang.org/ruby.git
Cloning into 'ruby'...
fatal: repository 'https://git.ruby-lang.org/ruby.git/' not found
```

**Comment by mame** [comment]
Fixed and released rubyfarm-bisect 1.1.3.  Thanks!

**Comment by yahonda** [comment]
I have confirmed `rubyfarm-bisect` 1.1.3 works as expected. Thank you for the fix.

```
% rubyfarm-bisect -v
rubyfarm-bisect 1.1.3
% rubyfarm-bisect -g 02aadf1032d71a3f81653a955f2635c8853b0aca -b b563439274a402e33541f5695b1bfd4ac1085638 ruby -e 'p /re/.match?(nil)'
```
